app/assets/stylesheets/nollning.scss

```diff
@@ -0,0 +1,33 @@
+$top: "theme/top.svg";
+$pillars: "theme/pillars.svg";
+$stairs: "theme/stairs.svg";
+.roman-top {
+  border: 90px solid transparent;
+  padding-top: 25px;
+  -webkit-border-image: url($top) 40% 40% 40% 40% repeat;
+  -o-border-image: url($top) 40% 40% 40% 40% repeat;
+  border-image: url($top) 40% 40% 40% 40% repeat;
+}
+.roman-pillars {
+  border-top: 20px solid transparent;
+  border-bottom: 20px solid transparent;
+  border-left: 110px solid transparent;
+  border-right: 110px solid transparent;
+  -webkit-border-image: url($pillars) 40% 40% 40% 40% repeat;
+  -o-border-image: url($pillars) 40% 40% 40% 40% repeat;
+  border-image: url($pillars) 30% 37% 30% 37% repeat;
+  margin-left: 10px;
+  margin-right: 10px;
+}
+.roman-footer {
+  border-top: 110px solid transparent;
+  border-bottom: 20px solid transparent;
+  border-left: 110px solid transparent;
+  border-right: 110px solid transparent;
+  -webkit-border-image: url($pillars) 40% 40% 40% 40% repeat;
+  -o-border-image: url($pillars) 40% 40% 40% 40% repeat;
```

Comment:
The hound is right, partly. Vendor prefixes are ugly and they make me feel dirty but they are sometimes necessary.

I don't know about border-image but in general you should use the "macro" or "mixin" or whatever they call it feature of SASS/SCSS to make browser prefixes nicer to deal with.